packages/react-inbox/src/components/Messages2.0/Message.tsx

```diff
@@ -44,7 +44,7 @@ const UnreadIndicator = styled.div<{ read?: IInboxMessagePreview["read"] }>(
 );
 
 const ClickableContainer = styled.a(({ theme }) => {
-  const primaryColor = theme.brand?.colors?.primary;
+  const primaryColor = theme.brand?.colors?.primary ?? "#9121c2";
```

Comment:
what was the behavior before this?

Reply:
if a brand can't be loaded it seems to render black so this is just a fallback